sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -234,6 +234,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val IN_MEMORY_TABLE_SCAN_STATISTICS_ENABLED =
+    buildConf("spark.sql.inMemoryTableScanStatistics.enable")
```

Comment:
nit: `.enabled`?

Reply:
I also did the same mistake. :)
We cannot change the name because this is already used configuration name.

Reply:
There are more instance `.enable`. We may change them together.